src/client/components/ConsentsCommunicationCard.tsx

```diff
@@ -45,18 +37,18 @@ const communicationCardHeadingContainer = (image?: string) => css`
   display: flex;
   flex-direction: column;
   justify-content: flex-end;
-  padding: ${space[6]}px ${space[3]}px ${space[2]}px ${space[3]}px;
-
+  padding: 14px ${space[3]}px 14px ${space[3]}px;
```

Comment:
do we have an import for 14px?

Reply:
No, any hardcoded value should not have a design system import